test/spoof/spoof.go

```diff
@@ -47,6 +46,9 @@ const (
 	// TODO(tcnghia): These probably shouldn't be hard-coded here?
 	istioIngressNamespace = "istio-system"
 	istioIngressName      = "istio-ingressgateway"
+	// Name of the temporary http header that is added to http.Request to indicate that
+	// it is a SpoofClient.Poll request. This header is removed before making call to backend.
+	pollRequest = "POLL_REQUEST"
```

Comment:
Don't we want this to look like canonical header? `Kn-Poll-Request`, e.g.?
In addition name it `do-not-trace` or something, since that is the purpose?

Reply:
I am not sure if I have addressed the comment correctly. Partly because I didn't completely understand the header you were looking for. Please let know any changes required.

Reply:
Eh what I meant, is that doesn't look like a HTTP header. At least no a canonical one (e.g. `Content-Length`, or `Host`). What I recomment is using naming it something like `X-Knative-Dev-Poll-Request`.

Reply:
Yeah, I see what you did. Works for me. (`X-` prefix might be a good touch, but this is test, so meh).